src/components/pages/Footer.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components'; 
+import {Container} from './PagesStyles'
+
+const FooterText = styled.div`
+   background-color: #313539;
+   color: #acaeaf;
+   font-size: 0.7rem;
+   margin-top: 3rem;
+   padding: 1rem;
+`;
+
+const FooterContainer = styled(Container)`
+text-align: center;
+     > a {
+         color: #acaeaf;
+         
+     }
+`;
+
+const Footer = () => {
+	return (
+		<FooterText>
```

Comment:
have a good practice with indentation

Reply:
I thought I checked everything. Ok, I will do it. 
..and I don't have to rely too much on the 'prettier now' extension because it is not working as expected to be.
Could be this due to styled-components? I noticed that also the autocompletion during my typing is not working.
Anyway, I will pay more attention to it.
Thank you very much